grafana/k6-studio
# feat: crash reporter settings

## Description

This PR depends on https://github.com/grafana/k6-studio/pull/365. It adds a new option to the Settings dialog that allows users to opt out of k6 Studio sending error/crash reports to Sentry.



## How to Test

- manually throw an error anywhere in the application that is triggered on user click (e.g. toggle theme)
- toggle the setting and check that the error event is not sent when the setting is disabled



## Checklist



## Screenshots (if appropriate):

![Screenshot 2024-12-30 at 11 04 28 AM](https://github.com/user-attachments/assets/a7bbd705-45af-40be-956c-4eacd7b9139f)

## Related PR(s)/Issue(s)
